wandb_logger.py

```diff
@@ -19,6 +23,92 @@
 wandb.init(project='catlearn', config={})
 
 
+def compute_kge_metrics(
+    cache: RelationCache[NodeType, ArrowType],
+    triplets: List[Tuple[
```

Comment:
Can't we use type aliases? Reminds me a C++ template errors spat by the compilers x)

Reply:
On a side note: triplets looks to only require `Iterable`, which would make the function a bit more general

Reply:
Hahaha :D
Well the thing is even though this type definition is long (thanks to Python's typing syntax...) it's only used once, so not a fan of using a type alias just for this.
But I can abide if it reminds you too much of work xD

Reply:
Don't bother :-) My eyes will keep hurting anyway from day-to-day job